### DISCUSSION TITLE
Two finger tap

### DISCUSSION THREAD

**Discussion by steveruizok** [pmndrs/use-gesture#400] [Q&A] [answered]
Hey all, thanks for the wonderful library. We're using it in [tldraw](https://tldraw.com)!

I'm trying to implement a two-finger/three-finger taps for undo and redo, similar to what you may have used in procreate or other apps for the iPad. Before I start trying to implement this on my own, I was wondering if anyone has worked on this before? Or if there is anything in the library that could help me in recognizing these events?

**Comment by dbismut** [comment, marked as answer]
Hey @steveruizok, thanks for the kind words, I'm a big fan of your work btw ;)

There's no easy way to do this out of the box, although this is definitely something I could work on since the whole multi fingers detection is there. In the meantime you could do something like this:

https://github.com/pmndrs/use-gesture/discussions/295

But it uses swipe instead of tap, which I understand is not ideal. Give me some time, I'll figure something out, possibly introducing a new hook like `useTap`.

**Reply by LPink777** [reply]
Use two fingers touchend.but down prop is true
